drivers/flash/flash_rpc/flash_rpc_common.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/logging/log.h>
+#include <nrf_rpc/nrf_rpc_ipc.h>
+#include <nrf_rpc_cbor.h>
+
+LOG_MODULE_REGISTER(FLASH_RPC, CONFIG_FLASH_RPC_LOG_LEVEL);
+
+#ifdef CONFIG_FLASH_RPC_SYS_INIT
```

Comment:
rather than doing this, exclude the file if the Kconfig is not enabled from cmake

Reply:
Yeah makes sens I just copied how bt_rpmsg_common had done it.